<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Creature</class>
 <widget class="QDialog" name="Creature">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>384</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>899</width>
    <height>384</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>899</width>
    <height>384</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>RomanT</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="leftButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resurce.qrc">
     <normaloff>:/Consol/Button2.png</normaloff>:/Consol/Button2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="rightButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resurce.qrc">
     <normaloff>:/Consol/Button1.png</normaloff>:/Consol/Button1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="Name">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="Descriptions">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Descriptions</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Enemy">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Enemy</string>
   </property>
  </widget>
  <widget class="QLabel" name="Hints">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>271</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Hints</string>
   </property>
  </widget>
  <widget class="QLabel" name="TitleD">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="TitleE">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rubius Icons LTProtect Accent</family>
    </font>
   </property>
   <property name="text">
    <string>Text of enemy</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="TitleH">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Test of Hints</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Page1">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>299</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>P1</string>
   </property>
  </widget>
  <widget class="QLabel" name="Pic">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>80</y>
     <width>221</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string>Picture</string>
   </property>
  </widget>
  <widget class="QLabel" name="lSubtitle">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>RomanT</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kind of...</string>
   </property>
  </widget>
  <widget class="QLabel" name="Sublist">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>181</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>RomanT</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SubList</string>
   </property>
  </widget>
  <widget class="QLabel" name="List">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>371</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>260</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resurce.qrc">
     <normaloff>:/Consol/Home.png</normaloff>:/Consol/Home.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Resurce.qrc"/>
 </resources>
 <connections/>
</ui>
